demo/finder/DbFinder.php

```
@@ -10,16 +10,12 @@ public function __construct(DbInterface $db)
-     * @Inject
-     */
+    #[Inject]
     public function setDb(DbInterface $db)
     {
     }
 
-    /**
-     * @Inject
-     */
+    #[Inject]
     public function setSorter(Sorter $sorter, Sorter $sorte2)
```

PHPMD (2.15.0)

Avoid unused parameters such as '$sorter'. (Unused Code Rules)

(UnusedFormalParameter)